shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/StopDetailsDepartures.kt

```diff
@@ -112,6 +112,13 @@ data class StopDetailsDepartures(val routes: List<PatternsByStop>) {
         val stopName: String
     )
 
+    /*
+    Whether the upcoming trip with the given id is cancelled in the upcoming departures
+     */
+    fun tripIsCancelled(tripId: String): Boolean {
```

Comment:
If we're touching the iOS code in this PR anyway, we may as well call this from the iOS side, too.

Reply:
Good call!